int16 direction
int16 velocity
int16 rotation
---
bool receive_data
